src/vulkan-renderer/wrapper/mesh_buffer.cpp

```diff
@@ -59,13 +63,25 @@ MeshBuffer::MeshBuffer(const wrapper::Device &device, VkQueue data_transfer_queu
 }
 
 MeshBuffer::MeshBuffer(const wrapper::Device &device, VkQueue data_transfer_queue,
+                       const std::uint32_t data_transfer_queue_family_index, const VmaAllocator vma_allocator,
```

Comment:
`data_transfer_queue` and `data_transfer_queue_family_index` are never used.

Reply:
Not just there: it's a general issue in the code.
I will deal with this in another PR: https://github.com/inexorgame/vulkan-renderer/issues/238.